Deploy/stacks/UK-building-retrofit/StackDeployment/Stack-manager/inputs/config/services/vis-mapbox.json

```diff
@@ -0,0 +1,23 @@
+{
+    "ServiceSpec": {
+        "Name": "vis-mapbox",
+        "TaskTemplate": {
+            "ContainerSpec": {
+                "Image": "ghcr.io/cambridge-cares/dtvf-base-image:latest",
```

Comment:
This is hopelessly outdated. Is this deliberate? I thought you had upgraded everything to the latest TWA vis framework?

Reply:
Apologies, I copied the wrong visualization backup. I’ve now uploaded the version using TWA VF 4.6.0 in commit [4b17b6014b2f81e5f29eeb93a7c99e4cb941202f] and commit [e62212b3beebc69c3143e987516269629e6b716b]